source/documentation/doc-Release_Notes/topics/ref-ovirt-4.5.2.adoc

```diff
@@ -0,0 +1,102 @@
+=== Red Hat Virtualization 4.4 SP 1 Batch Update 2 (ovirt-4.5.2)
+
+
+
+==== Bug Fix
+
+These bugs were fixed in this release of Red Hat Virtualization:
+
+BZ#link:https://bugzilla.redhat.com/1955388[1955388]::
+Previously, the Manager could start a virtual machine with a Resize and Pin NUMA policy on a host that does not have an equal number of physical sockets to the number of NUMA nodes it has. As a result, the wrong pinning was assigned to the policy. With this release, the Manager does not allow the virtual machine to be scheduled on such a virtual machine, making the pinning correct based on the algorithm.
+
+BZ#link:https://bugzilla.redhat.com/2097558[2097558]::
+Previously, running engine-setup did not necessarily renew OVN certificates when they were close to expiration or expired. With this release, OVN certificates are always renewed by engine-setup when needed.
```

Comment:
I'm not sure what "necessarily" means. Does it mean engine-setup sometimes renewed OVN certificates?  If so, it would be clearer to replace it with "always". If engine-setup never renewed OVN certificates, then you do not need "necessarily".